blog/district-c-collaboration/index.md

```diff
@@ -0,0 +1,201 @@
+challenging. It helped us to reflect on our needs and better understand them by
+clear formulation.
+
+## The big event
+
+Student groups learned what operating in a modern global corporate workplace is
+like. They gathered input, applied design thinking principles to empathize,
+identified opportunities for growth, and suggested sound solutions, which were
+then presented during the final **pitch meeting** held over Zoom.
+
+![](images/hitachi_crew.png)
+
+**Pitch meeting** is a presentation where an individual or a team presents a
+business idea or concept to potential investors, stakeholders, or clients to
+secure funding, support, or interest in the project. The presentation usually
+includes an overview of the product or service and the proposed solution.
+
+## Examples of solutions
+
+Each group presented its assessment of the opportunities and provided practical
+recommendations. Fresh perspectives and solutions were appreciated, as they
+logically tied to the identified areas for improvement. At the same time, the
+Hitachi Energy team offered practical feedback, rooting in business operations,
+experience, and practice.
+
+Students focused mainly on the nature of the interactions between the Technical
+Writers and the Subject Matter Experts (SME).
+
+They then explored many valid questions:
+
+- What is the most efficient way to get SME insights and build product
+  knowledge?
+- Are multiple team members often asked the same questions?
+
+One suggestion was to invite all SMEs to a single session to get all their
+answers. Alternatively, this could be resolved asynchronously through Teams or a
+survey.\
+\
+Personally, District C inspired me to use Microsoft Loop, allowing me to engage
+multiple Subject Matter Experts quickly. I can reuse my Loop notes in bigger
+collections. I can embed them in MS Teams chats, channels, Outlook e-mails, and
+meeting invites. The same content appears in multiple places and can be edited
+instantly by me and the SMEs involved.
+
+Other recommendations included:
+
+- Conducting dedicated workshops to get product insights
+- Incorporating short videos to distribute knowledge and information
+- Using Scrum Master's advice to optimize the collaboration between the Tech
+- Writers and the Dev team
+- Establishing a more formalized review process
+
+Though we have not used every recommendation, we gained a clearer picture of our
+needs and possibilities. The challenge allowed for the analysis of how we
+operate on a daily basis, and it emphasized the strategic and business
+objectives that are key to the effectiveness of the processes we run in Krakow
+and for the global organization.
+
+As part of continuous improvement, I regularly reevaluate my approach to
+documentation process and collaboration. Even a year later, I still actively use
+the good ideas I picked up during the District C project.
+
+![](images/group_photo.jpg) **Happy participants at the Hitachi Energy Office in
+Krakow.**
```

Comment:
Paweł zasugerował wcześniej Kraków. Jeśli wybierzesz polską nazwę to musisz to zmienić wszędzie. Jeśli zdecydujesz się na oficjalną międzynarodową nazwę miasta Krakow, to też pilnuj tego wszędzie, żeby nie było raz tak, a raz inaczej ;)

Reply:
ok